package/functions/funcs/uri.js

```diff
@@ -0,0 +1,20 @@
+module.exports = async (d) => {
+  const code = d.command.code;
+  const inside = d.unpack();
+  const err = d.inside(inside);
+  if (err) return d.error(err);
+  let [type,text=""] = inside.splits; 
+  if(!type) d.error(':x: Pls provide a Type(encode/decode)')
```

Comment:
Provide 'return' keyword, so code won't be continously executed after error

Reply:
okay done forgot it sry